src/modes/hints/client/activate.js

```diff
@@ -46,6 +48,19 @@ const activators = {
     target.focus()
     return 'Reset'
   },
+  openMultipleLinksInBackgroundTabs: (event, target) => {
+    mouseEvent(target, 'click', { ctrlKey: !isMac, metaKey: isMac })
+    if (SAKA_PLATFORM === 'firefox') {
+      backgroundOpenLink('openLinkInBackgroundTab', target)
+    }
+    hideHints()
```

Comment:
Need to hide the current hint before re-showing them all because the element under the current hint will be considered "invisible" and a hint won't be rendered for it.